### PR TITLE
application: serial_lte_modem: Doc GNSS service configs

### PR DESCRIPTION
**Pull request by junqingzou**
Add description of the CONFIG items for GNSS services

Signed-off-by: Jun Qing Zou <jun.qing.zou@nordicsemi.no>

### CODE REVIEW

**Review by FrancescoSer — APPROVED** [review]

File: applications/serial_lte_modem/doc/slm_description.rst [hunk 1]
```diff
@@ -179,17 +179,27 @@ Check and configure the following configuration options for the sample:
 
    This option enables additional AT commands for using the SMS service.
 
-.. option:: CONFIG_SLM_GPS - GPS support in SLM
+.. option:: CONFIG_SLM_GNSS - GNSS support in SLM
 
-   This option enables additional AT commands for using GPS service.
+   This option enables additional AT commands for using GNSS service.
```
Comment (peknis) [inline applications/serial_lte_modem/doc/slm_description.rst:184]:
```suggestion
   This option enables additional AT commands for using the GNSS service.
```

File: applications/serial_lte_modem/doc/slm_description.rst [hunk 2]
```diff
@@ -179,17 +179,27 @@ Check and configure the following configuration options for the sample:
...
 
-.. option:: CONFIG_SLM_SUPL_SERVER - SUPL server
+.. option:: CONFIG_SLM_AGPS - nRF Cloud A-GPS support in SLM
 
-   This option specifies the SUPL server to use for retrieving SUPL A-GPS data.
+   This option enables additional AT commands for using nRF Cloud A-GPS service.
 
-.. option:: CONFIG_SLM_SUPL_PORT - SUPL server port
+   This option is not selected by default.
```
Comment (peknis) [inline applications/serial_lte_modem/doc/slm_description.rst:190]: You can have this right on the next line, without a blank line. -> "It is not selected by default.

File: applications/serial_lte_modem/doc/slm_description.rst [hunk 3]
```diff
@@ -179,17 +179,27 @@ Check and configure the following configuration options for the sample:
...
+.. option:: CONFIG_SLM_GNSS - GNSS support in SLM
 
-   This option enables additional AT commands for using GPS service.
+   This option enables additional AT commands for using GNSS service.
 
-.. option:: CONFIG_SLM_SUPL_SERVER - SUPL server
+.. option:: CONFIG_SLM_AGPS - nRF Cloud A-GPS support in SLM
 
-   This option specifies the SUPL server to use for retrieving SUPL A-GPS data.
+   This option enables additional AT commands for using nRF Cloud A-GPS service.
```
Comment (peknis) [inline applications/serial_lte_modem/doc/slm_description.rst:188]:
```suggestion
   This option enables additional AT commands for using the nRF Cloud A-GPS service.
```

File: applications/serial_lte_modem/doc/slm_description.rst [hunk 4]
```diff
@@ -179,17 +179,27 @@ Check and configure the following configuration options for the sample:
...
 
-.. option:: CONFIG_SLM_SUPL_PORT - SUPL server port
+   This option is not selected by default.
+
+.. option:: CONFIG_SLM_PGPS - nRF Cloud P-GPS support in SLM
+
+   This option enables additional AT commands for using nRF Cloud P-GPS service.
```
Comment (peknis) [inline applications/serial_lte_modem/doc/slm_description.rst:194]:
```suggestion
   This option enables additional AT commands for using the nRF Cloud P-GPS service.
```
And the same with the "It is not selected by default".

File: applications/serial_lte_modem/doc/slm_description.rst [hunk 5]
```diff
@@ -179,17 +179,27 @@ Check and configure the following configuration options for the sample:
...
+
+   This option is not selected by default.
+
+.. option:: CONFIG_SLM_CELL_POS - nRF Cloud cellular positioning support in SLM
 
-   This option specifies the port to use for the specified SUPL server.
+   This option enables additional AT commands for using nRF Cloud cellular positioning service.
```
Comment (peknis) [inline applications/serial_lte_modem/doc/slm_description.rst:200]:
```suggestion
   This option enables additional AT commands for using the nRF Cloud Cellular Positioning service.
```
I think the service name must be capitalized. And again the same with the next line.

**Review by peknis — CHANGES_REQUESTED** [review]

File: applications/serial_lte_modem/doc/slm_description.rst (hunk 2 above)
Comment (junqingzou) [reply applications/serial_lte_modem/doc/slm_description.rst:190]: The empty line and sentence is the same as previous CONFIG items.
Remove all empty lines but keep the sentence no change to align with other CONFIG items not in this PR.

**Review by peknis — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

**Pull request merged**